src/usps_processor/core.clj

```diff
@@ -27,3 +27,6 @@
   (immutant.util/at-exit stop-importer)
   (info "Starting api server")
   (api/-main))
+
+(defn test-main [& args]
+  nil)
```

Comment:
Hm... littering the core namespace with a test function. I'm not sure how I feel about it.